common.blocks/menu/menu.js

```diff
@@ -131,6 +112,23 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
         return null;
     },
 
+    /** @override **/
+    _onFocus : function() {
```

Comment:
Good catch :+1: Seems that no.
I think I've fixed it by second commit. Take a look.